### ISSUE TITLE
CompreFace Security Concerns and Regulatory Compliance

### ISSUE THREAD

**Issue by AnkurChatter** [exadel-inc/CompreFace#769]
Hi @pospielov 
Had a couple of questions regarding security and regulatory compliance - 

1. How are the face-images stored by CompreFace docker container ? Can anyone access the docker resource and get those images out ?
2. Are the images stored in docker container encrypted, if so which encryption methodology ?
3. What kind of regulatory compliance does CompreFace adhere to ? I meant compliance like GDPR, Sarbanes-Oxley Act, HIPAA, PCI DSS and CCPA.

**Comment by pospielov** [comment]
1. Face images are stored in the Postgres database as usual binary data. So they are not encrypted in any way. Default CompreFace configuration doesn't open Postgres port, so only those who have access to docker containers(e.g. using ssh) can access the database. We recommend deploying CompreFace in a secure virtual network, so nobody has access to containers directly.
2. Answered the previous point
3. Regulatory compliances are important for us and we do everything to be compliant with them. We do not store or send any information to our servers, except anonymous system usage information, and you can disable even it. 
However, all compliances are depends on you. E.g. one of the requirements is to notify users of how we store and manage data. We have this information in our repository. But you are responsible for notifications of your users. Also, you are responsible for changing the default password in the database. 
So, in short, I believe CompreFace can be integrated into the system and the system can be compliant after this. But this is your responsibility to make it right.

Also, very important to notice, that there is a `save_images_to_db` variable. If it is set to `false`, CompreFace won't save any images to DB:
https://github.com/exadel-inc/CompreFace/blob/master/docs/Configuration.md
